CMakeLists.txt

```diff
@@ -8,6 +8,14 @@ set(CMAKE_VERBOSE_MAKEFILE ON)
 option(BUILD_BINDINGS "Build the python bindings" ON)
 option(${PROJECT_NAME}_CXX_WARNINGS "Compile C++ with warnings" ON)
 option(EXTERNAL_HIGHFIVE "Use HighFive from external source" OFF)
+option(USE_DOUBLE "Use doubles instead of floats" OFF)
```

Comment:
I feel like the project name should be added to the configuration option. Seems pretty generic otherwise.

Reply:
Will add for more clarity. Thanks